frontend/app/globals.css

```
@@ -1,12 +1,11 @@
-@import url("https://fonts.googleapis.com/css2?family=Outfit:wght@300;400;500;600&family=Shippori+Mincho+B1:wght@400;500;600;700;800&display=swap");
 @import "tailwindcss";
 @import "tw-animate-css";
 @import "shadcn/tailwind.css";
 
 :root {
-  --app-font-body: "Outfit", "Hiragino Sans", "Yu Gothic UI", system-ui, sans-serif;
+  --app-font-body: var(--font-outfit, "Outfit"), var(--font-noto, "Noto Sans SC"), "Hiragino Sans", system-ui, sans-serif;
   --app-font-mono: "SFMono-Regular", "IBM Plex Mono", monospace;
-  --app-font-display: "Shippori Mincho B1", "Hiragino Mincho ProN", Georgia, serif;
+  --app-font-display: var(--font-shippori, "Shippori Mincho B1"), "Hiragino Mincho ProN", Georgia, serif;
```

Stylelint (17.7.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)